apps/src/eval/eval.js

```diff
@@ -151,6 +151,36 @@ Eval.init = function(config) {
     // base's studioApp.resetButtonClick will be called first
     var resetButton = document.getElementById('resetButton');
     dom.addClickTouchEvent(resetButton, Eval.resetButtonClick);
+
+    if (Blockly.contractEditor) {
+      Blockly.contractEditor.registerTestHandlers(function (exampleBlock) {
```

Comment:
I wonder if this would be more clear as two functions - registerTestHandler and registerTestResetHandler.